rasa/core/interpreter.py

```diff
@@ -211,7 +211,10 @@ def __init__(self, endpoint: EndpointConfig = None) -> None:
 
         params = {"token": self.endpoint.token, "text": text, "message_id": message_id}
 
-        url = urljoin(self.endpoint.url, "/model/parse")
+        if self.endpoint.url.endswith("/"):
```

Comment:
Why no using the same logic for both cases? 
So either
```
if not self.endpoint.url.endswith("/"):
    self.endpoint.url = self.endpoint.url + "/"

url = urljoin(self.endpoint.url, "model/parse")
```
or
```
if self.endpoint.url.endswith("/"):
    url = self.endpoint.url + "model/parse"
else:
    url = self.endpoint.url + "/model/parse"  
```

Reply:
Thanks. Resolved